google/resource_container_cluster_test.go

```diff
@@ -712,6 +712,7 @@ resource "google_container_cluster" "with_node_config" {
 			foo = "bar"
 		}
 		tags = ["foo", "bar"]
+		preemptible = true
```

Comment:
I'm not really sure- I did this one sort of as an experiment, and it definitely doesn't catch everything since it just checks that state matches remote, but not that those are actually the values we want (I actually had a bug slip through once because of this). Right now I've been using `TestCheckResourceAttrSet` to check that the correct values are in state for what I care about but I don't know if that actually checks that I did it with a .Set.